lib/src/payment_intent.dart

```diff
@@ -6,22 +6,27 @@ class PaymentIntent {
   String paymentMethodId;
   String returnURL;
   String clientSecret;
+  bool isSavingPaymentMethod;
 
   PaymentIntent({
     this.paymentMethod,
     this.paymentMethodId,
     this.returnURL,
     @required this.clientSecret,
+    this.isSavingPaymentMethod,
   });
 
   Map<String, dynamic> toJson() {
     final Map<String, dynamic> data = new Map<String, dynamic>();
     if (this.paymentMethod != null) {
       data['paymentMethod'] = this.paymentMethod.toJson();
     }
-    if (this.paymentMethodId != null) data['paymentMethodId'] = this.paymentMethodId;
+    if (this.paymentMethodId != null)
+      data['paymentMethodId'] = this.paymentMethodId;
     if (this.returnURL != null) data['returnURL'] = this.returnURL;
     if (this.clientSecret != null) data['clientSecret'] = this.clientSecret;
+    if (this.isSavingPaymentMethod != null)
```

Comment:
is this attribute actually read by the native counterparts? can you link me to it?

Reply:
Do you mean native code in tipsi-stripe library?
if so here are the lines: 
[iOS](https://github.com/jonasbark/flutter_stripe_payment/blob/eaa2961d248e27fa94036591d3fa3904c4286244/ios/Classes/TPSStripeManager.m#L1003)
[Android](https://github.com/jonasbark/flutter_stripe_payment/blob/f0fb958d2471941c9b75d73ee5cf629de4449a3d/android/src/main/java/com/gettipsi/stripe/StripeModule.java#L465)